api/lib/domain/usecases/find-user-campaign-participation-overviews.js

```diff
@@ -0,0 +1,8 @@
+module.exports = async function findUserCampaignParticipationOverviews({ userId, states, campaignParticipationOverviewRepository, page }) {
+  if (states && states.length > 0) {
+    const newStates = Array.isArray(states) ? states : [states];
```

Comment:
Proposition : 

```suggestion
    const newStates = [].concat(states);
```
devrait pouvoir faire l'affaire.

Reply:
```suggestion
    const newStates = [...states];
```
doit même être possible

Reply:
Si states n'est pas un tableau, tu ne pourra pas utiliser la notation `...` dessus je pense.